services/src/main/java/uk/gov/moj/sdt/services/mbeans/SdtManagementMBean.java

```diff
@@ -36,7 +36,12 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.context.annotation.Scope;
```

Comment:
These two imports are not used.  Remove them if they are not needed.

Reply:
removed